draft-ietf-privacypass-architecture.md

```diff
@@ -199,6 +199,42 @@ does not reply to the Origin's challenge with a new request.
 ~~~
 {: #fig-overview title="Privacy pass redemption and issuance protocol interaction"}
 
+The end-to-end flow for Privacy Pass involves three different types of contexts:
+
+- Redemption context: The interactions and set of information shared
+between the Client and Origin.
+- Issuance context: The interactions and set of information shared
+between the Client, Attester, and Issuer.
+- Attestation context: The interactions and set of information shared between
+the Client and Attester only, for the purposes of attesting the vailidity of the Client.
+
+The privacy goals of Privacy Pass are oriented around unlinkability based on these
+contexts. In particular, Privacy Pass aims to achieve Client unlinkability. This
+means that given two redemption (or issuer) contexts, the Origin (or Issuer) cannot
+determine if both redemption (or issuance) contexts correspond to the same Client
```

Comment:
The redemption/issuer split here is a bit confusing to read. I know it would be redundant to explain both, but it may be easier to read.

Reply:
ACK -- I'll try to unwrap that in a followup commit.

Reply:
OK, I split unlinkability into three distinct properties: Origin-Client, Issuer-Client, and Attester-Origin unlinkability. How does this read now?